actors/builtin/miner/monies.go

```diff
@@ -124,17 +124,11 @@ func InitialPledgeForPower(qaPower abi.StoragePower, baselinePower abi.StoragePo
 // may be slightly lower than the true amount. Computing vesting here would be
 // almost always redundant since vesting is quantized to ~daily units.  Vesting
 // will be at most one proving period old if computed in the cron callback.
-func VerifyPledgeRequirementsAndRepayDebts(rt Runtime, st *State) abi.TokenAmount {
+func RepayDebts(rt Runtime, st *State) abi.TokenAmount {
```

Comment:
Not sure about this name, just trying out something much simpler

Reply:
I don't think this needs to be exported, which means it probably moves to `miner_actor.go` as `repayDebts`.